### PR TITLE
main/gnupg1: security upgrade to 1.4.23

### PR DESCRIPTION
**Pull request by tcely**

### CODE REVIEW

**Comment by ncopa** [comment]
Are there any other issues than CVE-2018-12020, which was fixed with 7a15f5277c131da11081bea9a0077891e58ac0fb?

File: main/gnupg1/APKBUILD [hunk 1]
```diff
@@ -12,10 +12,11 @@ makedepends="curl-dev libassuan libksba-dev libgcrypt-dev libgpg-error-dev
 subpackages="$pkgname-doc"
 provides="gnupg=$pkgver-r$pkgrel"
 replaces="gnupg"
-source="ftp://ftp.gnupg.org/gcrypt/gnupg/gnupg-$pkgver.tar.bz2
-	0001-gpg-Sanitize-diagnostic-with-the-original-file-name.patch"
+source="https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-$pkgver.tar.bz2"
 
 # secfixes:
+#   1.4.23-r0:
+#     - CVE-2017-752
```
Comment (ncopa) [inline main/gnupg1/APKBUILD]: I guess you mean [CVE-2017-7526](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-7526)

**Comment by ncopa** [comment]
i fixed secfixes comment and commit message. thanks!

**Comment by tcely** [comment]
@ncopa Thanks. I'm not sure how I managed to drop that last digit in the CVE number.

**Pull request closed without merging**